src/migrations/20201029230535-create-orders.js

```diff
@@ -14,6 +13,13 @@ module.exports = {
       },
       userId: {
         type: Sequelize.UUID,
+        references: {
+          model: {
+            tableName: 'Users',
+            schema: 'public',
```

Comment:
curious, what does this schema thing do / is it necessary?

Reply:
A schema is a collection of DB objects, in our case, we only have `public` which is the PostgreSQL default, so all of our tables belong to this schema. I thought it was necessary but I looked into the docs and it uses `public` by default if not provided so I'll remove it from the references.

https://www.sqlshack.com/a-walkthrough-of-sql-schema/ - just the first paragraph
https://sequelize.org/master/class/lib/dialects/abstract/query-interface.js~QueryInterface.html#instance-method-createSchema

Reply:
thanks!